src/Maintenance/Maintenance.Test/Maintenance.Test.csproj

```diff
@@ -14,6 +14,10 @@
     <Folder Include="SessionRecords\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Azure.ResourceManager.Compute" Version="1.6.0" />
```

Comment:
we do not support track2 sdk, please use track1 sdk instead, also add reference for network sdk

Reply:
Yes, I was little confused while adding module dependencies. I have now fixed it.